modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -252,4 +262,235 @@ void EditorSceneFormatImporterBlend::get_import_options(const String &p_path, Li
+				*r_err = TTR("Path lacks a Blender binary.");
+			}
+			return false;
+		}
+		pipe = pipe.replace_first("Blender ", "");
+		int pp = pipe.find(".");
+		if (pp == -1) {
+			if (r_err) {
+				*r_err = TTR("Path lacks a a Blender binary.");
+			}
+			return false;
+		}
+		String v = pipe.substr(0, pp);
+		int version = v.to_int();
+		if (version != 3) {
+			if (r_err) {
+				*r_err = TTR("This Blender installation is an older, unsupported version (not 3.0+)");
+			}
+			return false;
+		}
+
+		return true;
+	}
+}
+
+bool EditorFileSystemImportFormatSupportQueryBlend::is_active() const {
+	bool blend_enabled = GLOBAL_GET("filesystem/import/blender/enabled");
+
+	String blender_path = EDITOR_GET("filesystem/import/blender/blender3_path");
+
+	if (blend_enabled && !_test_blender_path(blender_path)) {
+		// Intending to import Blender, but blend not configured.
+		return true;
+	}
+
+	return false;
+}
+Vector<String> EditorFileSystemImportFormatSupportQueryBlend::get_file_extensions() const {
+	Vector<String> ret;
+	ret.push_back("blend");
+	return ret;
+}
+
+void EditorFileSystemImportFormatSupportQueryBlend::_validate_path(String p_path) {
+	String error;
+	bool success = false;
+	if (p_path == "") {
+		error = TTR("Path is empty.");
+	} else {
+		if (_test_blender_path(p_path, &error)) {
+			success = true;
+			if (auto_detected_path == p_path) {
+				error = TTR("Path to Blender installation is valid (Autodetected).");
+			} else {
+				error = TTR("Path to Blender installation is valid.");
+			}
+		}
+	}
+
+	path_status->set_text(error);
+
+	if (success) {
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("success_color"), SNAME("Editor")));
+		configure_blender_dialog->get_ok_button()->set_disabled(false);
+	} else {
+		path_status->add_theme_color_override("font_color", path_status->get_theme_color(SNAME("error_color"), SNAME("Editor")));
+		configure_blender_dialog->get_ok_button()->set_disabled(true);
+	}
+}
+
+bool EditorFileSystemImportFormatSupportQueryBlend::_autodetect_path(String p_path) {
+	if (_test_blender_path(p_path)) {
+		auto_detected_path = p_path;
+		return true;
+	}
+	return false;
+}
+
+void EditorFileSystemImportFormatSupportQueryBlend::_path_confirmed() {
+	confirmed = true;
+}
+
+void EditorFileSystemImportFormatSupportQueryBlend::_select_install(String p_path) {
+	blender_path->set_text(p_path);
+	_validate_path(p_path);
+}
+void EditorFileSystemImportFormatSupportQueryBlend::_browse_install() {
+	if (blender_path->get_text() != String()) {
+		browse_dialog->set_current_dir(blender_path->get_text());
+	}
+
+	browse_dialog->popup_centered_ratio();
+}
+
+bool EditorFileSystemImportFormatSupportQueryBlend::query() {
+	if (!configure_blender_dialog) {
+		configure_blender_dialog = memnew(ConfirmationDialog);
+		configure_blender_dialog->set_title(TTR("Configure Blender Importer"));
+		configure_blender_dialog->set_flag(Window::FLAG_BORDERLESS, true); // Avoid closing accidentally .
```

Comment:
Sounds like the comment is meant for the next line?

Reply:
Not kinda, since no borders, no close button either.